todo_app/app.py

```diff
@@ -1,31 +1,45 @@
-from flask import Flask, render_template, request
 
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
-from todo_app.data import session_items
+from todo_app.data import trello_items as trelloItems
+import os
 
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+listtodo = os.environ.get('LISTTODO')
+listdoing = os.environ.get('LISTDOING')
+listdone = os.environ.get('LISTDONE')
 
 @app.route('/')
 def index():
-    items = session_items.get_items()
-    return render_template("index.html", items = items)
+    items = trelloItems.get_items()
+    return render_template('index.html', items = items, todo = listtodo , doing =listdoing, done = listdone)
 
-@app.route('/newitem', methods=['POST'])
-def create_new():
 
-    #call add item
-    data = request.form.get('newitem')
-    session_items.add_item(data)
-    #re render
-    items = session_items.get_items()
-    #line changed to use flask funtionality
-    #return redirect('/')
-    return render_template("index.html", items = items)
+@app.route('/items/new', methods=['POST'])
+def add_item():
+    title = request.form['title']
+    trelloItems.add_item(title)
+    return redirect(url_for('index'))
 
 
+@app.route('/items/<id>/complete')
```

Comment:
You can make this accept a POST request in a similar way to the `/items/new` route i.e. add `methods=['POST']`.

You'll then need to use forms in your HTML.
Instead of doing
```html
<a href="{{ url_for('complete_item', id=item.id) }}" class="badge badge-success">Close Item</a>
```
you can do
```html
<form action="{{ url_for('complete_item', id=item.id) }}" method="POST">
    <button type="submit" class="btn btn-success">Close item</button>
</form>
```
To send a POST request